react/components/Modal/index.js

```diff
@@ -66,6 +69,8 @@ Modal.propTypes = {
   closeOnEsc: PropTypes.bool,
   /** Close the modal on overlay click (default true) */
   closeOnOverlayClick: PropTypes.bool,
+  /** Show the close icon on upper right corner (default true) */
+  showCloseIcon: PropTypes.bool,
```

Comment:
should be `hideCloseIcon`if default is `true`.

Reply:
I copied the same name that 'react-responsive-modal' uses. I think it's better to use names the devs are used to instead of mapping same functionalities to different names.

I think it is better to leave it that way. What do you think ?